zbus/src/utils.rs

```diff
@@ -17,6 +17,7 @@ pub(crate) fn padding_for_n_bytes(value: usize, align: usize) -> usize {
 /// type alias.  This is currently required because the macros for properties expect a Result
 /// return value, but the macro-generated `receive_` functions need to refer to the actual
 /// type without the associated error.
+#[doc(hidden)]
```

Comment:
This is not supposed to be private either. Read the docs. :)

Reply:
actually, I think we could hide ResultAdapter, it's an implementation detail of the proxy macro. No?

Reply:
hmm.. yeah. so probably you were right to hide it (we've other apis too that are hidden and only `pub` because of macros).